tests/src/test/scala/whisk/core/limits/ActionLimitsTests.scala

```diff
@@ -60,6 +60,31 @@ class ActionLimitsTests extends TestHelpers with WskTestHelpers {
 
     behavior of "Action limits"
 
+    it should "limit the length of HTTP Req/Resp Body for --verbose to 1000 bytes" in withAssetCleaner(wskprops) {
```

Comment:
This test is in the wrong file. ActionLimitTests are for action limits related to the RestAPI. Should move to WskBasicUsageTests.scala.